modules/oadp-release-notes-1-1-6.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-6_{context}"]
+= OADP 1.1.6 release notes
+
+The OADP 1.1.6 release notes lists any new features, resolved issues and bugs, and known issues.
+
+[id="new-features1.1.6_{context}"]
+== New features
+
+// This version of OADP is a service release. No new features are added to this version.
+
+[id="resolved-issues1.1.6_{context}"]
+== Resolved issues
+
+.Support the standardized STS configuration flow using Operator Lifecycle Manager (OLM) and Cloud Credential Operator (CCO) for OADP in OCP 4.14
```

Comment:
It's not supported in OADP 1.1.6. @weshayutin please correct me if I'm wrong

Reply:
Removed - apologies, misread the ticket and did not see the closed - Duplicate status